Readme.md

```diff
@@ -3,16 +3,18 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E714)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* **[Form1.cs](./CS/AlwaysShowCellHints/Form1.cs) (VB: [Form1.vb](./VB/AlwaysShowCellHints/Form1.vb))**
-<!-- default file list end -->
-# How to show tooltips for grid cells even if their content is completely visible
 
+# Data Grid for Windows Forms - How to show tooltips for grid cells even if their content is completely visible
 
-<p>This is an example for <a href="https://www.devexpress.com/Support/Center/p/A2472">Showing a hint for a grid cell even if its content is completely visible</a> Knowledge Base article. It demonstrates how to display custom tooltips for grid cells.</p>
-
-<br/>
+The [ToolTipController](https://docs.devexpress.com/WindowsForms/DevExpress.Utils.ToolTipController) component allows you to manage tooltips for DevExpress Windows Forms controls. 
```

Comment:
Can we make the phrase "to manage tooltips" more specific? For example, would it be technically correct to say "manage tooltip behavior"? Feel free to suggest an alternative as well.

Reply:
```suggestion
The [ToolTipController](https://docs.devexpress.com/WindowsForms/DevExpress.Utils.ToolTipController) component allows you to manage tooltip behavior for DevExpress Windows Forms controls. 
```